.eslintrc.js

```diff
@@ -29,6 +29,6 @@ module.exports = {
     'react-native/no-single-element-style-arrays': 2,
     'no-console': 0,
     disallowMultipleVarDecl: 0,
-    'react/react-in-jsx-scope': 'off',
+    'react/react-in-jsx-scope': 3,
```

Comment:
why?

Reply:
I wrote it wrong here, I read that it should be disabled. If using React v17 and above and React Native, we can safely disable the rule in the eslint configuration file.